phd.h

```diff
@@ -79,7 +79,7 @@
 
 #define APPNAME _T("PHD2 Guiding")
 #define PHDVERSION _T("2.6.10")
-#define PHDSUBVER _T("dev3")
+#define PHDSUBVER _T("dev3_VarDelay")

```

Comment:
need to revert this before we merge
```suggestion
#define PHDSUBVER _T("dev3")
```

Reply:
Yes, for sure.